panda.config.ts

```diff
@@ -113,6 +136,21 @@ const globalCss = defineGlobalStyles({
     userSelect: 'none',
     cursor: 'auto',
   },
+  // Sets default link color in recipes/modal color
+  '.modal__root': {
```

Comment:
`.modal__root` and `.modal__actions` are build artifacts, so we don't want to code against them. Is there a different we can apply these styles?

Reply:
Descendant selectors won't work here (I tried), so the only way to replace it that I could think of was to add the styles directly to each <a> tag. Since there's a lot, it may be hard remember to be consistent with it, and there would be a lot of instances to replace. 

I do know this isn't a good method, but since it get's put in the @base layer, it can be overridden by any recipe variants. Also, this behaviour is at least documented: https://panda-css.com/docs/concepts/slot-recipes#targeting-slots

Another good option would be to use `defineParts`. This would still require adding it directly to each <a> tag, but would making writing it easier, and would allow any changes to apply to all of the links.
https://panda-css.com/docs/concepts/slot-recipes#defineparts
I was a bit unsure on this, since there was some talk about deprecating it a few years ago, and it isn't really documented, but it's been a while since then, and I don't think they are planning on deprecating it. I believe some component libraries use it under the hood, and I found an issue where the maintainers specifically said to use it over `sva`

Reply:
Thanks for describing what options we have. 

Since the slot class is not type safe, and defineParts just takes us deeper into proprietary slot functionality, I think the better approach is to apply a recipe directly to each link. That has the benefit of co-location, simplicity, and moving fully away from descendant selectors. 

Since this does require a lot of small changes, I recommend doing it in a separate PR. I'll leave it up to you whether you want to do it after this PR, or circle back later on.

Reply:
Sounds good, I'll circle back to this later on.